lib/msf/ui/console/command_dispatcher/creds.rb

```diff
@@ -490,86 +484,20 @@ def creds_search(*args)
           human_val, #private type
           jtr_val
         ]
-      else
-        core.logins.each do |login|
-          service = framework.db.services(id: login.service_id).first
-          # If none of this Core's associated Logins is for a host within
-          # the user-supplied RangeWalker, then we don't have any reason to
-          # print it out. However, we treat the absence of ranges as meaning
-          # all hosts.
-          if host_ranges.present? && !host_ranges.any? { |range| range.include?(service.host.address) }
-            next
-          end
-
-          row = [ service.host.address ]
-          row << origin
-          rhosts << service.host.address
-          row << build_service_info(service)
-
-          matched_cred_ids << core.id
-          public_val = core.public ? core.public.username : ""
-          private_val = core.private ? core.private.to_s : ""
-          if truncate && private_val.to_s.length > 87
-            private_val = "#{private_val[0,87]} (TRUNCATED)"
-          end
-          realm_val = core.realm ? core.realm.value : ""
-          human_val = core.private ? core.private.class.model_name.human : ""
-          if human_val == ""
-            jtr_val = "" #11433, private can be nil
-          else
-            jtr_val = core.private.jtr_format ? core.private.jtr_format : ""
-          end
-
-          row += [
-            public_val,
-            private_val,
-            realm_val,
-            human_val,
-            jtr_val
-          ]
-          tbl << row
-        end
       end
     end
-    if mode == :delete
-      result = framework.db.delete_credentials(ids: matched_cred_ids)
-      delete_count = result.size
-    end
 
     if output_file.nil?
-      print_line(tbl.to_s)
+      print_line(tbl.to_s) if tbl
```

Comment:
Shouldn't we be printing out a warning of some type here if `tbl` isn't valid as there will be no way to provide output at this point?